chasm/lib/scheduler/proto/v1/request_response.proto

```diff
@@ -0,0 +1,62 @@
+syntax = "proto3";
+
+package temporal.server.chasm.lib.scheduler.proto.v1;
+
+option go_package = "go.temporal.io/server/chasm/lib/scheduler/gen/schedulerpb;schedulerpb";
+
+import "temporal/api/workflowservice/v1/request_response.proto";
+
+message CreateScheduleRequest {
+    temporal.api.workflowservice.v1.CreateScheduleRequest request = 1;
```

Comment:
I want to establish a pattern where we call these fields `frontend_request` and in the response` frontend_response` just to clarify a bit more what they are and avoid the `request.Request` stutter in the code.